docs/source/jit.rst

```diff
@@ -531,13 +531,252 @@ Python-defined Constants
 Debugging
 ~~~~~~~~~
 
-Print things
+Interpreting Graphs
+    TorchScript uses a static single assignment (SSA) intermediate representation
+    (IR) to represent computation. The instructions in this format consist of
+    ATen operators and other primitive operators, including control flow
```

Comment:
Many users don't know what ATen is. Can you at least add a tiny comment like "ATen (the C++ backend of PyTorch) operators